interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java

```diff
@@ -449,10 +451,10 @@ private void runTest(TestCases testCase) throws Exception {
         if (fullStreamDecompression) {
           okBuilder.enableFullStreamDecompression();
         }
+        InternalOkHttpChannelBuilder.setStatsEnabled(okBuilder, false);
         builder = okBuilder;
       }
-      // Disable the default census stats interceptor, use testing interceptor instead.
-      io.grpc.internal.TestingAccessor.setStatsEnabled(builder, false);
+      // Use testing interceptor instead.
```

Comment:
Make some mention of `setStatsEnabled()` here, so it is clearer the two lines are related. It'd probably be fine to duplicate the `builder.intercept(createCensusStatsClientInterceptor());` in both the okhttp and netty paths to keep the two lines together. But separate is fine, as long as we point out they are related.